_studio/mfx_lib/decode/vp9/src/mfx_vp9_dec_decode_hw.cpp

```diff
@@ -522,7 +522,13 @@ mfxStatus VideoDECODEVP9_HW::DecodeHeader(VideoCORE* core, mfxBitstream* bs, mfx
 
     if (   par->mfx.FrameInfo.FourCC == MFX_FOURCC_P010
         )
+    {
```

Comment:
it could be simply:
```
par->mfx.FrameInfo.Shift = mfxU16(par->mfx.FrameInfo.FourCC == MFX_FOURCC_P010);
```

Reply:
In my opinion concrete values "0" and "1"are better for understanding, especially for people who see source code the first time, so I think we can keep this code in this style or we can change to 1-line style, for example:

par->mfx.FrameInfo.Shift = (par->mfx.FrameInfo.FourCC == MFX_FOURCC_P010) ? 1 : 0;

Reply:
I think oneliner is better in this situation. I'm OK with your suggestion to use operator ?:

Reply:
What about y210?

Reply:
VP9 HW decoder doesn't support y210, only nv12, ayuv, p010, p016.